cli/reference/blocks/terramate.md

```diff
@@ -4,3 +4,46 @@ description: Learn how to configure a Terramate project using the terramate bloc
 ---
 
 # The `terramate` block
+
+Use the `terramate` block to define project-wide configurations in the project's root. The `terramate` block is defined in the `terramate.tm.hcl` file and overrides any defaults by Terramate.
+
+## Arguments
+
+- `required_version`: specify the Terramate version to be used inside a project. More details about this attribute are [here](../../projects/configuration.md#the-terramaterequired_version-attribute).
+- `config`: define project-wide configurations in this block.
+    - `config.git`: [configure](../../projects/configuration.md#the-terramateconfiggit-block) git integration
+    - `config.generate`: [configure](../../projects/configuration.md#the-terramateconfiggenerate-block) code generation feature
+    - `config.run`: [configure](../../projects/configuration.md#the-terramateconfiggenerate-block) the `terramate run` command or set environment variables for it
+    - `config.cloud`: [configure](../../projects/configuration.md#the-terramateconfigcloud-block) the default Terramate Cloud organization name
```

Comment:
Cloud is the block, the `organization` attribute is used to configure the Terramate Cloud Organization used in this project